mail/utils.py

```diff
@@ -85,3 +85,21 @@ def generate_mailgun_response_json(response):
             "message": response.reason
         }
     return response_json
+
+
+def filter_recipient_variables(text):
```

Comment:
Could you write a test or two for this, I think it's technically covered by other tests, but I don't think the expected behavior is being asserted.